### DISCUSSION TITLE
Support OrderedDict with Tree?

### DISCUSSION THREAD

**Discussion by findmyway** [FedeClaudi/Term.jl#104] [Ideas]
In some cases, I'd like to customize the order of the children in a Tree. So it would be great if we can support OrderedDict or something similar.

**Comment by FedeClaudi** [comment]
Tree should work with `AbstractDict` as inputs: https://github.com/FedeClaudi/Term.jl/blob/fe8525ffcbc9f8f38b56465ed415a844795f2480/src/tree.jl#L159

Are `OrderedDict` subtypes of `AbstractDict`?

**Comment by findmyway** [comment]
Ah, I see. Yes, it works. I should have tested it first. Thanks!

**Comment by FedeClaudi** [comment]
No problem, I can add more explicit info to the docs!
